docs/submitting.md

```diff
@@ -54,10 +54,26 @@ Authors wishing to publish software deemed out of scope for JOSS have a few opti
 
 While we are happy to review submissions in standalone repositories, we also review submissions that are significant contributions made to existing packages. It is often better to have an integrated library or package of methods than a large number of single-method packages.
 
-### Questions? Open an issue to ask
+### Questions? Open an issue to ask.
 
 Authors wishing to make a pre-submission enquiry should [open an issue](https://github.com/openjournals/joss/issues/new?title=Pre-submission%20enquiry) on the JOSS repository.
 
+## Conflict of Interest policy for authors
+
+An author conflict of interest (COI) arises when an author has financial, personal, or other interests that may influence their research or the interpretation of its results. In order to maintain the integrity of the work published in JOSS, we require that authors disclose any potential conflicts of interest at submission time.
+
+### Policy
+
+**Disclosure:** All authors must disclose any potential conflicts of interest related to the research in their manuscript, including financial, personal, or professional relationships that may affect their objectivity. This includes any financial relationships, such as employment, consultancies, honoraria, stock ownership, or other financial interests that may be relevant to the research.
+
+**Acknowledgement:** Authors should acknowledge all sources of financial support for the research and include a statement indicating whether or not the sponsor had any involvement in the study design, data collection, analysis, or interpretation of the work.
```

Comment:
@danielskatz – I've remove 'results' here and changed it for 'work'. It's very likely that the software will have been developed as part of some broader research program so it seems OK to allude to that here?

Reply:
I don't think these specifics (study design, data collection, analysis, or interpretation) make sense for JOSS still.  How about just "Authors should acknowledge all sources of financial support for the work and include a statement indicating whether or not the sponsor had any involvement in it."